bbb/services.py

```diff
@@ -399,6 +405,19 @@ def handlePending(self, data, msg):
                 self.bbb_db.deleteBuildRequest(our_task.buildrequestId)
             return
 
+        # The script that the Bridge relies on to send build finished events
+        # requires that "product" be set in the Buildbot Properties. If it's
+        # not provided by the Task, we shouldn't accept the job because it's
+        # unlikely that we'll notice when its Build finishes. See bug 1195751
+        # for additional background.
+        if not product:
+            log.info("Task %s has no product set, refusing to create BuildRequest", taskid)
```

Comment:
Not AFAIK - You can't add any custom status or comments when canceling a task, only when resolving it.